elyra/pipeline/pipeline.py

```diff
@@ -314,6 +326,79 @@ def __eq__(self, other: "GenericOperation") -> bool:
     def _validate_range(self, value: str, min_value: int = 0, max_value: int = sys.maxsize) -> bool:
         return int(value) in range(min_value, max_value)
 
+    @staticmethod
+    def get_valid_volume_mounts(
```

Comment:
Should this be an instance method `get_valid_volume_mounts()` to make it easier for consumers of the `Pipeline` object? Right now consumers have to make two calls - one to fetch the list and one to coerce it into the desired data structure. If there was a consumer that did require access to the raw data, it would still be able to gain access via `get_component_parameter(...)`

Reply:
Well this is a result of the issue with potentially using these methods from the `describe` command, for example. The processors access their parameter values from the `Operation` object, but `describe` accesses parameters from the `PipelineDefinition` object (and each have their own method of providing that value). I haven't fully checked yet to see what the potential implications of using a `Pipeline` object instead of a `PipelineDefinition` object in `describe` would be yet, but I'm not sure it would be as straightforward as it sounds 😄

Reply:
Can we move these to the base `Operation` class in preparation for their general availability?